scripts/build/companion_libs/220-ncurses.sh

```diff
@@ -49,7 +49,7 @@ do_ncurses_for_host() {
     # (i.e. in simple cross or native): ncurses may not be needed for
     # host, but we still need them on build to produce 'tic'.
     case "${CT_TOOLCHAIN_TYPE}" in
-        native|cross)   return 0;;
+        native)   return 0;;
```

Comment:
As a side note, we should probably add -I/-L options to FOR_BUILD variables too, but so far it hasn't bitten us :)